static/lang/re-en.json

```diff
@@ -1739,6 +1739,9 @@
                 "Prompt": "Select a clan weapon."
             },
             "Cleric": {
+                "CommunalHealing": {
+                    "Note": "When you cast the <em>heal</em> spell to heal a single creature, choose another willing living creature within range of your <em>heal</em> to regain [[/r (@item.rank)[vitality,healing]]] Hit Points."
```

Comment:
What does this look like in the chat log?

Reply:
<img width="285" alt="Screen Shot 2024-01-27 at 11 23 02 PM" src="https://github.com/foundryvtt/pf2e/assets/1904698/1a93a253-98b2-40cb-99f1-0affdeeea40a">

Do you want me to remove the vitality type?

Reply:
You can specify a `shortLabel` option now in `@Damage` expression.

Reply:
Since it's healing, it's `[[/r ]]` rather than `@Damage[]`.

Reply:
I went ahead and removed the vitality type.